inst/topics/july4th.R

```diff
@@ -25,5 +25,6 @@ c(
   "Liberty",
   "Flag",
   "Patriot",
-  "Summer"
+  "Summer",
+  "testline"
```

Comment:
The change in this file is probably unintentional? Please resolve in your branch and push or let me know if you'd prefer I update. Then I could accept the changes to the football card.